community/configuration/src/main/java/org/neo4j/configuration/ExternalSettings.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.configuration;
+
+import java.util.function.Function;
+
+import org.neo4j.graphdb.config.Configuration;
+import org.neo4j.graphdb.config.Setting;
+
+/**
+ * This class holds settings which are used external to the java code. This includes things present in the
+ * configuration which are only read and used by the wrapper scripts. By including them here, we suppress warning
+ * messages about Unknown configuration options, and make it possible to document these options via the normal methods.
+ *
+ * Be aware that values are still validated.
+ */
+@SuppressWarnings( "unused" )
+public class ExternalSettings implements LoadableConfig
```

Comment:
Could this be provided by the `server` module or something else higher up the stack? `command-line`?